notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<h2>Download and preprocess the data</h2><p> </p><p>This section shows how to download the flower images, use the tf.data API to create a data input pipeline, and split the data into training and validation sets.</p><p> </p><p>Move the part about <code>TFRecords</code> into the relevant section below ("Read images and labels from TFRecords"). In general, [it's best to avoid parenthetical statements](https://developers.google.com/style/parentheses).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>